recipes-dev-platform/images/genivi-dev-platform.bbappend

```diff
@@ -1,5 +1,11 @@
-# Don't ship the standard GDB UI
-IMAGE_INSTALL_remove = "packagegroup-gdp-hmi"
+# Don't ship the standard GDB, the browser and weston
+IMAGE_INSTALL_remove = " \
+    packagegroup-gdp-hmi \
+    packagegroup-gdp-browser \
+    wayland-ivi-extension \
+    packagegroup-abstract-component-p1 \
+    weston \
```

Comment:
Yeah, weston would sometimes start befor AppMan and then AppMan wouldn't be able to start.